src/sentry/api/serializers/models/group.py

```diff
@@ -460,11 +466,17 @@ def _get_seen_stats(
         perf_issues = [
             group for group in item_list if GroupCategory.PERFORMANCE == group.issue_category
         ]
+        generic_issues = [
+            group for group in item_list if GroupCategory.PROFILE == group.issue_category
```

Comment:
This should probably be changed to be if it's not errors or perf and not hard code profiling issues.

Reply:
Good catch.